# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1744142713 -7200
# Parent  379562d072d217a9db675a2d9e8f3dcc3ab57505
Misc: Reduce minimum width of Message Pane (vertical view, see bug 1815407).

diff --git a/mail/base/content/about3Pane.xhtml b/mail/base/content/about3Pane.xhtml
--- a/mail/base/content/about3Pane.xhtml
+++ b/mail/base/content/about3Pane.xhtml
@@ -354,9 +354,9 @@
     </div>
   </div>
   <hr is="pane-splitter" id="messagePaneSplitter"
       resize-id="messagePane"
-      collapse-width="300"
+      collapse-width="200"
       collapse-height="100" />
 #include templates/messagePaneTemplate.inc.xhtml
   <message-pane id="messagePane" class="collapsed-by-splitter"></message-pane>
   <xul:browser id="accountCentralBrowser" hidden="true"/>
diff --git a/mail/themes/shared/mail/about3Pane.css b/mail/themes/shared/mail/about3Pane.css
--- a/mail/themes/shared/mail/about3Pane.css
+++ b/mail/themes/shared/mail/about3Pane.css
@@ -160,9 +160,9 @@ body {
   box-sizing: border-box;
   background-color: var(--layout-background-1);
   /* Matches the collapse-height and collapse-width on the splitter. */
   min-height: 100px;
-  min-width: 300px;
+  min-width: 200px;
   overflow: auto;
   display: flex;
   flex-direction: column;
 }
